packages/atomic/src/components/commerce/search-box-suggestions/atomic-commerce-search-box-query-suggestions/atomic-commerce-search-box-query-suggestions.tsx

```diff
@@ -0,0 +1,137 @@
+import {
+  SearchBox,
+  Suggestion,
+  CommerceEngine,
+  loadQuerySuggestActions,
+} from '@coveo/headless/commerce';
+import {
```

Comment:
We must not do this kind of import, it's very problematic. 😄  

I realize this is a copy paste from classic search box suggestion component, which I did not catch while doing my refactor. But we should not spread it more.